components/announcement_bar/cloud_announcement_bar/user_limit_announcement_bar.tsx

```diff
@@ -28,7 +28,6 @@ import {t} from 'utils/i18n';
 import AnnouncementBar from '../default_announcement_bar';
 
 type Props = {
```

Comment:
if we remove this logic from here, is there any reason to keep these files? I think the mere purpouse of this announcement bar is notify about the user limit, so I guess we can get rid of them.